pennylane/interfaces/autograd.py

```diff
@@ -1,302 +1,302 @@
-# Copyright 2018-2021 Xanadu Quantum Technologies Inc.

```

Comment:
Did anything actually get changed in this file, or was this a git problem?  I had some git problems like this recently on my PR's.  If possible, we don't want to do this as it eliminates git blame and the history of the file.

Reply:
We had to change it to make the device specific tests but at the end it has been restored as it was.